cmd/translatesfx/README.md

```diff
@@ -11,43 +11,57 @@ in some cases to produce OpenTelemetry Collector configs that are fully
 functional and comparable to the original Smart Agent config. However, 
 this tool is designed to produce only a reasonably accurate approximation of 
 the final, production Otel config that would replace a Smart Agent config. It is 
-not designed to, and cannot, produce a drop-in replacement Otel config for a
-Smart Agent one in all cases.
+not designed to, and cannot, in all cases produce a drop-in replacement Otel
+config for a Smart Agent one.
 
-This tool aims to remove a lot of the drudgery from migrating to Otel 
-from Smart Agent, but any config produced by this tool should be carefully 
+Instead, this tool aims to remove a lot of the bookkeeping from migrating to Otel 
+from Smart Agent; any config produced by this tool should be carefully 
 evaluated and tested before being put into production.
```

Comment:
Is there another term to use besides "bookkeeping"? Used in this context, I think it could be confusing to some users.

Reply:
Replaced "bookeeping" and replaced all offending Otels.